README.md

```diff
@@ -15,6 +15,14 @@ However, there seems to be no curation for all these awesome HCI + DL/ Human-AI
 | Year |  Paper |
 |------|-------------------------------|
 | 2019 | [Guidelines for Human-AI Interaction](https://www.microsoft.com/en-us/research/uploads/prod/2019/01/Guidelines-for-Human-AI-Interaction-camera-ready.pdf) |
+| 2019 | [AI-Mediated Communication: How the Perception that Profile Text was Written by AI Affects Trustworthiness](http://www.mauricejakesch.com/pub/chi2019__ai_mc_camera_ready.pdf) |
+| 2019 | [Designing Theory-Driven User-Centric Explainable AI](https://www.ashrafabdul.com/pdf/xai-framework-preprint-chi2019.pdf) |
+| 2019 | [Will You Accept an Imperfect AI? Exploring Designs for Adjusting End-user Expectations of AI Systems](https://www.microsoft.com/en-us/research/uploads/prod/2019/01/chi19_kocielnik_et_al.pdf) |
+| 2019 | [ATMSeer: Increasing Transparency and Controllability in Automated Machine Learning](https://arxiv.org/pdf/1902.05009.pdf) |
+| 2019 | [In a Silent Way: Communication Between AI and Improvising Musicians Beyond Sound](https://arxiv.org/pdf/1902.06442.pdf) |
+| 2019 | [Exploring Factors that Influence Connected Drivers to (Not) Use or Follow Recommended Optimal Routes](https://www.andrew.cmu.edu/user/jseering/papers/Seering%20et%20al%202019%20Social%20Chatbots.pdf) |
```

Comment:
Seems like the paper link is not correct for this paper, It links to _The Effect of Explanations and Algorithmic Accuracy on Visual Recommender Systems of Artistic Images". This paper also seems related though.